test/spec/controllers/tasksCtrlSpec.js

```diff
@@ -1,12 +1,17 @@
 'use strict';
 
 describe('Tasks Controller', function() {
-  var $rootScope, shared, scope, user, ctrl;
+  var $rootScope, shared, scope, user, User, ctrl;
 
   beforeEach(function() {
-    user = specHelper.newUser();
```

Comment:
If I leave it that way then I have to call controllers with this notation `$controller('RootCtrl',  {$scope: scope, User: {user: user}});` . I just decided to clean up the notation at the top and keep it consistent with other tests files. That way, we don't have to call `User: {user: user}` at every controller initialization.